cinema/migrations/0001_initial.py

```diff
@@ -145,5 +171,8 @@
                     ),
                 ),
             ],
+            options={
+                "unique_together": {("movie_session", "row", "seat")},
```

Comment:
The `unique_together` constraint in the Ticket model ensures that no two tickets can have the same movie session, row, and seat combination. This is crucial for maintaining data integrity.